src/shared/components/ProfessionCard.tsx

```diff
@@ -3,14 +3,13 @@ import { Card, CardHeader, Avatar } from '@material-ui/core';
 import styled from 'styled-components';
 import { ProfessionKey } from '../../data/ITProfessions';
 
-const CategoryCardContainer = styled(Card)`
+const ProfessionCardContainer = styled(Card)<{ $isClickable: boolean }>`
```

Comment:
I know it's not part of your PR, but maybe we could rename the `Container` suffixes in this file to `Styled` prefixes? For example, `ProfessionCard` rather than `ProfessionCardContainer`. 
It may seem very picky on my part, but I have a good reason to suggest it: by convention `container` is a React component that has logic, state etc. as opposed to the so called "dumb" component that has no logic. When you are used to this, seeing 4 container components in one file is a bit confusing :)